### ISSUE TITLE
Have `ad_command` create a test file

### ISSUE THREAD

**Issue by arthur-shaw** [lsms-worldbank/adodown#11]
Currently, `ad_command` creates for the user-specified command:

- ado file in `src/ado`
- md file in `src/mdhlp`

In addition, create a test do file in `src/tests`

This would be a nice way to nudge towards best practices.

**Comment by kbjarkefur** [comment]
Agreed, the way I would like to set the test files up do require `rootenv`/`reprun` setup. So I am kind of waiting for that instead of figuring out how a to configure a default 

```
if "`c(username)'" == "wb462869" {
    global clone "C:/Users/wb462869/github/labeller"
}
```

**Comment by kbjarkefur** [comment]
Implemented in 93dcc50e311cbd60239dd3e081471daed763e7cb. Will close when published.
